src/components/KAlert/KAlert.vue

```diff
@@ -116,6 +116,7 @@
 
       <KButton
         v-if="dismissType === 'button'"
+        class="white-space-nowrap"
```

Comment:
question:
Shouldn't a KButton, by default, have `white-space: nowrap`?

suggestion: I think this rule should just be added directly to KButton instead?

Also, let's not keep adding utility classes. Again, as we move towards an actual design system, all of these utility classes will have to be refactored out of Kongponents (and utilize different, targeted classes) so let's not add more in the meantime if not absolutely necessary.

Reply:
Yeah you're right, better handle that from KButton itself. Removed the newly added utility class too (keep forgetting we don't want to add new ones at all).